src/icons/fileIcons.ts

```diff
@@ -467,7 +467,7 @@ export const fileIcons: FileIcons = {
       ],
     },
     { name: 'lib', fileExtensions: ['lib', 'bib'] },
-    { name: 'ruby', fileExtensions: ['rb', 'erb'] },
+    { name: 'ruby', fileExtensions: ['rb', 'erb', 'rbs'], fileNames: ['.ruby-version'] },
```

Comment:
Could you please format the code with prettier? The pipeline is complaining about it. Everything else looks good to me 👍

Reply:
Hi @PKief 

Thanks for your quick response. 👍 

However, I'm not a professional JavaScript delelopers.... so my computer isn't setup with `js` dev tools. 🥲

I tried to install the VSCode `prettier` and `prettier-eslint` plugins. But it shows something wrong with `Prettier` at the bottom status bar of VSCode. 

I guess it may depend on a local `NodeJS` runtime or library or something. Then I install `nodejs` and run `npm install`, but **it fails to complete.....** and I think I can't fix it in a short time.

<img width="670" alt="image" src="https://github.com/PKief/vscode-material-icon-theme/assets/63459097/858154c7-3458-450a-9c25-1ec8979253f4">

Could you help format it? If not convenient, please tell me where I'm wrong to invoke the `prettier` plugin, thanks!

Reply:
Hm okay I'm not aware of this error and can't help to fix that at the moment. I expect the formatter wants that the code just has some line breaks like this:

{ 
  name: 'ruby', 
  fileExtensions: ['rb', 'erb', 'rbs'],
  fileNames: ['.ruby-version'],
},

Can you try this out?